homeassistant/components/light/xiaomi_miio.py

```diff
@@ -323,9 +402,39 @@ def delayed_turn_off_timestamp(countdown: int,
 
         return None
 
+    # pylint: disable=no-self-use
+    async def async_reminder_on(self):
+        """Enable the eye fatigue notification."""
+        return
```

Comment:
`raise NotImplementedError`

Reply:
Is raising `NotImplementedError` correct here? Some devices (classes) doesn't support the feature / service. In this case the service call should be ignored. All xiaomi_miio lights uses the same DATA_KEY. If somebody calls `light.xiaomi_miio_reminder_on` (without an entity_id parameter) all entities will be addressed and the method needs to be accessible.

Reply:
But why wouldn't the service call filter them out?

```python
for device in devices:
    if not hasattr(device, 'async_reminder_on'):
        continue
    await device.async_reminder_on()
```

Reply:
This is cool. Thanks!